packages/@aws-cdk/aws-eks/README.md

```diff
@@ -639,6 +639,29 @@ mypod.node.addDependency(sa);
 new cdk.CfnOutput(this, 'ServiceAccountIamRole', { value: sa.role.roleArn })
 ```
 
+You can also add service accounts to existing clusters. 
+
+To do so, pass the `openIdConnectProvider` property when you import the cluster into the application.
+```ts
+// you can import an existing provider
+const provider = eks.OpenIdConnectProvider.fromOpenIdConnectProviderArn(this, 'Provider', 'OIDCProviderARN');
```

Comment:
Can you provide an actual provider ARN here? Would be nice to see exactly what it looks like instead of just `OIDCProviderARN`

Reply:
added an actual provider ARN